modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -527,3 +529,31 @@ function editConsentStatusFields($db, $user)
         }
     }
 }
+
+/**
+ * Handles the updating of candidate's date of birth.
+ *
+ * @param Database $db   database object
+ * @param User     $user user object
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editCandidateDOB($db, $user)
```

Comment:
Do you need to pass these as params? I think you could just use the singletons instead.

If you do need them, it would be good to add types:
```suggestion
function editCandidateDOB(\Database $db, \User $user): void
```